ballerina/Module.md

```diff
@@ -0,0 +1,184 @@
+* Make sure to go to the `Connections` tab in the CData Connect Server dashboard and set up any connection you need to work with the data sources. 
+You can use the **Connections** tab to configure a data source that contains the data you want to work with. 
+For more information on working with each data source, see [Data Sources](https://cloud.cdata.com/docs/Data-Sources.html).
+Use the **Connection Name** and **Data Source Name** to write your SQL queries.
+ 
+## Quickstart
+
+To use the CData Connect connector in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import connector and driver
+Import the following modules into the Ballerina project:
+```ballerina
+import ballerina/sql;
+import ballerinax/cdata.connect as cdata;      // Get the CData connector
+import ballerinax/cdata.connect.driver as _;   // Get the CData driver
+```
+
+### Step 2: Create a new connector instance
+Provide the `<username>`, `<password>` to initialize the Cdata Connect client connector. 
+Depending on your requirement, you can also pass optional properties and connection pool configurations during the client connector initialization. 
+For more information on connection string properties, see [Connection String Options](https://cdn.cdata.com/help/LHG/jdbc/Connection.htm).
+
+If you want to connect to CData Connect Cloud, you don't need to specify the `url`. 
+The default value will be `jdbc:cdata:connect:AuthScheme=Basic`.
+
+`<username>` is the email address of the user. For example `user@cdata.com`.
+`<password>` is a personal access token (PAT) that you generate from the **User Profile** page.
+
+```ballerina
+string user = "<username>";
+string password = "<password>";
+
+cdata:Client cdataClient = check new (user, password);
+```
+
+If you want to connect to CData Connect Server, you need to specify the `url` as `jdbc:cdata:connect:URL=http://<hostname>:<port>/rest.rsc;`.
+Replace the `<hostname>` and `<port>` with the information where your CData Cloud Connect Server is up and running.
+`<username>` is the username you use to login to CData Cloud Connect Server.
+`<password>` is the password you use to login to CData Cloud Connect Server.
+
+```ballerina
+string user = "<username>";
+string password = "<password>";
+string url = "jdbc:cdata:connect:URL=http://<hostname>:<port>/rest.rsc;";
+
+cdata:Client cdataClient = check new (user, password, url);
+```
+
+You can also define `<username>`, `<password>` and `<url>` as configurable strings in your Ballerina program.
+
+### Step 3: Invoke the connector operation
+1. Use the Ballerina CData Connect client connector to consume the CData Connect Cloud API. For more information on working with each data source, see [Data Sources](https://cloud.cdata.com/docs/Data-Sources.html). When you write SQL queries, be sure to specify the **Connection Name** and **Data Source Name** in the [Connections](https://cloud.cdata.com/docs/Connections.html) tab of the CData Cloud dashboard.
```

Comment:
We need to use a consistent name for the connector across the content. The `Ballerina CData Connect client connector` should change to `CData Connect Connector`.

Reply:
This is resolved now. Thank you.